modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss

```diff
@@ -37,6 +37,10 @@
 				.content-wrapper {
 					padding: 40px 24px;
 
+					.panel-header-link {
```

Comment:
I would not remove the box shadow if the Clay component already have it.

Reply:
Done! thank you!